_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,147 @@
+author: "@MattRowe18"
+image: 'img/carla.jpg'
+background: '/img/posts/2021-08-02/town10.png'
+---
+
+{% include youtube.html id="tWAg1fIDNCg" %}
+
+{% include release_button.html release_tag="0.9.13" %}
+
+The __0.9.13 release of CARLA__ is finally here, so read on to see how the new improvements will improve your workflow. 
+
+CARLA 0.9.13 brings a host of new API features and improvements for pedestrian and vehicle behaviour and the traffic manager to better mimic real world behaviour of cars and people.
+
+The most exciting new improvements come in the traffic manager. The traffic manager now offers __new API functionality__ to guide vehicles with a __user defined path__, defined by location waypoints or through route commands like left, right, straight, etc. __Vehicle lights__ can now be controlled by the traffic manager. 
+
+The CARLA garage now has a __brand new vehicle__, the __Volkswagen T2 2021__. All generation 1 and 2 cars now include opening doors that can interact with the environment through collisions, especially useful for training and testing autonomous vehicles in car parks or on tight city streets. 
+
+__Installation improvements:__ following a continued commitment to improve the useability of the CARLA simulator, the CARLA agents wheel is now __available through PyPi__ so can now be installed with pip. A new wheel for the core CARLA API library is now also available for Python version 3.9.
+
+The user can also now define custom animations and gestures for pedestrians through the python API. 
+ 
+
+- [__Traffic Manager__](#traffic-manager): New functions for user defined paths
+- [__New and improved vehicles__](#new-and-improved-vehicles): A brand new vehicle and opening doors with improved breaking and gear behaviour
+- [__Runtime texture updating__](#runtime-texture-updates): Runtime texture updating through the CARLA Python API
+- [__Python library upgrades__](#python-library-upgrades): carla-agents library now available through PyPi and new Python 3.9 wheels
+
+## Traffic Manager
+
+### User defined paths
+
+The traffic manager has been extended with some great new functionality to extend user control over the behaviour of vehicles in the traffic manager. The API now exposes new traffic manager functions that can be used to guide vehicles with a user-defined custom path.
+
+The `set_path` function allows a custom route to be defined using coordinates defined as CARLA Locations:
+
+```py
+path = [carla.Location(x=-506.696198, y=179.384308, z=0.038194),
+                    carla.Location(x=-504.745972, y=232.868927, z=0.039417)]
+traffic_manager.set_path(vehicle, path)
+```
+The `set_route` function allows a custom route to be defined using route commands:
+
+```py
+route = ["Right", "Straight", "Right", "Right", "Left", "Right"]
+ traffic_manager.set_route(vehicle, route)
+```
+
+![set_route](/img/posts/2021-11-07/follow_route.gif){:class="img-fluid rounded mx-auto d-block"}
+
```

Comment:
There are also the `get_next_action` and `get_all_actions` for users to know what will a TM vehicle do. this is useful to know they direction they'll take during an intersection